CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.1.4
+* Fixed a timezone issue with unit tests
+* Upgrade prgx to 0.11.1
+* Reworked Dockerfile - multi-stage, better extension configuration, added a Makefile with 'all' and 'run' targets
+* Improve README
```

Comment:
None of these are user facing and need not be in Changelog.

Reply:
plz define user facing...

who is the user?
define "facing"

besides I kind of disagree, at least those two a "user facing" - if i get this definition right

* Upgrade prgx to 0.11.1
* Reworked Dockerfile - multi-stage, better extension configuration, added a Makefile with 'all' and 'run' targets

Reply:
Why would SQL users or admins care about these?